src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -39,9 +47,13 @@
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: INDEX (must be a positive integer) "
             + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_NRIC + "NRIC] "
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
+            + "[" + PREFIX_BIKE + "BIKE] "
+            + "[" + PREFIX_LOANRATE + "LOANRATE] "
```

Comment:
LoanTime is not supposed to be editable.

Reply:
Apologies, I referenced the wrong line. This comment references line 56, which is 1 line below.